src/api/user.api.ts

```diff
@@ -23,8 +23,15 @@ router.get('/user/:id',
 );
 
 router.get('/', async (req, res) => {
-    const users = await userCtrl.getAllUsers();
-    res.json(users);
+    await userCtrl.getAllUsers().then(users => {
+        res.json(users);
+    }).catch(err => console.log(err));
+});
+
+router.get('/deletedUsers', async (req, res) => {
+    await userCtrl.getDeletedUsers().then(users => {
+        res.json(users);
+    }).catch(err => console.log(err));
```

Comment:
// REMARK: I assume that we should allow such cases to hoist and add global error handler
    // Default error handler: https://expressjs.com/en/guide/error-handling.html